### PR TITLE
[BACKEND][CPU] Make it buildable and runnable in a different environment

### PR DESCRIPTION
**Pull request by minjang**
For some reasons, my environment (Centos 9) needs some quick fixes to make the current triton-cpu buildable and runnable.

Build:
```
DEBUG=1 LLVM_INCLUDE_DIRS=$LLVM_BUILD_DIR/include LLVM_LIBRARY_DIR=$LLVM_BUILD_DIR/lib LLVM_SYSPATH=$LLVM_BUILD_DIR pip install -e python 
```

Running a tutorial (you need to fix the tutorial though, which I will commit as well):
```
TRITON_CPU_BACKEND=1 python3 python/tutorials/01-vector-add.py
```

### CODE REVIEW

File: third_party/cpu/backend/driver.py
Comment (minjang) [inline third_party/cpu/backend/driver.py:1]: This change is due to the auto-formatter (yapf).

Comment (ienkovich) [reply third_party/cpu/backend/driver.py:1]: How did you run auto-formatter? I get different results when following the pre-commit check from the CI workflow:
```
python3 -m pip install --upgrade pre-commit
python3 -m pre_commit run --all-files --verbose yapf
```
Changes in `compiler.py` match but I have fewer changes in `driver.py`

File: third_party/cpu/backend/driver.py [hunk 1]
```diff
@@ -176,7 +177,8 @@ def format_of(ty):
     arg_ptrs_list = ', '.join(f"&arg{i}" for i, ty in signature.items()) if len(signature) > 0 else ''
     kernel_fn_args = [i for i in signature.keys() if i not in constants]
     kernel_fn_args_list = ', '.join(f"arg{i}" for i in kernel_fn_args) if len(kernel_fn_args) > 0 else ''
-    kernel_fn_arg_types = (', '.join(f"{ty_to_cpp(signature[i])}" for i in kernel_fn_args) + ", " if len(signature) > 0 else '') + "uint32_t, uint32_t, uint32_t"
+    kernel_fn_arg_types = (', '.join(f"{ty_to_cpp(signature[i])}" for i in kernel_fn_args) +
+                           ", " if len(signature) > 0 else '') + "uint32_t, uint32_t, uint32_t"
```
Comment (minjang) [inline third_party/cpu/backend/driver.py:181]: Hm, looks like this is still changed by yapf.

Comment (ienkovich) [reply third_party/cpu/backend/driver.py:181]: This now matches what I get from the pre-commit run. Thanks!

**Review by ienkovich — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**